### DISCUSSION TITLE
A happy new year to all!

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#166] [General]
A happy new year to all!

thanks to you for making me spend a very interesting end of year 2021 with the discovery of BiblitoteQ.

I hope that the coming quarters will be as rich in improvements.

I have just seen the discussion with jerzyPL who manages a bilbiothèque of 25000 documents. I hope I will benefit from his feedback.

once again take care of yourself with I hope the exceptional programmer : textbrowser and BiblioteQ and take care of yourself and Happy New Year 2022.

Best consideration

**Comment by textbrowser** [comment]
Thanks for the kind words! I hope you and your family have an awesome 2022 as well!

**Comment by meteos77** [comment]
thank you also ; we are first on the project to have a good time : it's your hobby, mine is the computer management of the volunteers library :-)

**Reply by textbrowser** [reply]
If you can bring any happiness to the world, do it. :)

**Reply by meteos77** [reply]
we are in the same state of mind :-)
